packages/ot-cryptid/PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name="OneTrick-CRYPTID"
+pkgname=ot-cryptid
+pkgver=1.0.1
+pkgrel=1
+pkgdesc='FM drum synthesizer with a Yamaha DX-7 emulation sound engine'
+arch=(aarch64 x86_64)
+url='https://punklabs.com/ot-cryptid'
+license=(GPL3)
+depends=(gcc-libs glibc libglvnd libx11 libxcb libxcursor xcb-util-wm)
+makedepends=(alsa-lib cargo jack python rust)
+optdepends=(
+  'jack: for running the stand-alone app'
+  'clap-host: for loading the CLAP format plugin'
+  'vst3-host: for loading the VST3 format plugin'
+)
+groups=(clap-plugins pro-audio vst3-plugins)
+source=("https://punklabs.com/content/projects/$pkgname/downloads/$_name-$pkgver-Source.zip")
+sha256sums=('3856c2f7a3c47f66d8b7878dfe7704ba6d48e23f83db9ba137f1c625fef355fd')
+noextract=("$_name-$pkgver-Source.zip")
+
+prepare() {
+  mkdir -p $_name-$pkgver
+  bsdtar -xf "$srcdir"/$_name-$pkgver-Source.zip -C $_name-$pkgver
+}
+
+build() {
+  cd $_name-$pkgver
+  cargo xtask bundle onetrick_cryptid --release
```

Comment:
not sure if this changes when fetching crates in `prepare()`

Reply:
Looks like it works as is.

I added

```
  export RUSTUP_TOOLCHAIN=stable
  export CARGO_TARGET_DIR=target
```

there too for the benefit of AUR users.